tei_adapter/tei_id_extractor/src/main/scala/weco/catalogue/tei/id_extractor/database/RDSClientBuilder.scala

```diff
@@ -5,19 +5,17 @@ import scalikejdbc.{ConnectionPool, ConnectionPoolSettings}
 import uk.ac.wellcome.typesafe.config.builders.EnrichConfig._
 
 object RDSClientBuilder {
-  def buildDB(rdsClientConfig: RDSClientConfig): Unit = {
-
-    Class.forName("com.mysql.jdbc.Driver")
+  def buildDB(rdsClientConfig: RDSClientConfig): Unit =
     ConnectionPool.singleton(
       s"jdbc:mysql://${rdsClientConfig.host}:${rdsClientConfig.port}",
       user = rdsClientConfig.username,
       password = rdsClientConfig.password,
       settings = ConnectionPoolSettings(
         maxSize = rdsClientConfig.maxConnections,
-        connectionTimeoutMillis = 120000L
+        connectionTimeoutMillis = 600000L
```

Comment:
Since I can't work out milliseconds in my head, how about a Duration helper to make this clearer:

```scala
import scala.concurrent.duration._

connectionTimeoutMillis = (10.minutes).toMillis
```